docs/api.yml

```diff
@@ -0,0 +1,307 @@
+                        - type
+                        - content
+                      properties:
+                        group:
+                          type: string
+                          description: Name of the task
+                        subgroup:
+                          type: integer
+                          description: Id of the worker
+                        start:
+                          type: integer
+                          format: timestamp
+                          description: Start time of the task in unix timestamp format
+                        end:
+                          type: integer
+                          format: timestamp
+                          description: End time of the task in unix timestamp format when `type` is range
+                        type:
+                          type: string
+                          enum: [range, point]
+                        content:
+                          type: string
+                          description: Short info about the task
+                        title:
+                          type: string
+                          format: html
+                          description: Detailed information about the task
+                        style:
+                          type: string
+                          description: Style attribute to apply on the rendered task
+                  groups:
+                    type: array
+                    items:
+                      type: object
+                      required:
+                        - id
+                        - content
+                        - order
+                      properties:
+                        id:
+                          type: string
+                        content:
+                          type: integer
+                        order:
+                          type: integer
+  /job/{job_id}/cancel:
+    post:
+      summary: Cancel the job with the specified job_id
+      parameters:
+        - name: job_id
+          in: path
+          schema:
+            type: string
+            format: uuid
+          required: true
+      responses:
+        200:
+          description: Request processed
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  status:
+                    type: string
+  /queue/{queue_name}/empty:
+    post:
+      summary: Empty the queue with the specified queue_name
+      parameters:
+        - name: queue_name
+          in: path
+          schema:
+            type: string
+          required: true
+      responses:
+        200:
+          description: Request processed
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  status:
+                    type: string
+  /queue/{queue_name}/delete:
+    post:
```

Comment:
why post instead of delete?

Reply:
@dkusnik just documenting the state here ^^